apps/dev/pages/api/examples/protected.js

```diff
@@ -1,8 +1,8 @@
 // This is an example of to protect an API route
-import { getSession } from "next-auth/react"
+import { getServerSession } from "next-auth/react"
 
 export default async (req, res) => {
-  const session = await getSession({ req })
+  const session = await getServerSession({ req })
```

Comment:
`const session = await getServerSession({ req, res }, options)`

And we have to decide where options come from.